roles/installer/templates/deployments/web.yaml.j2

```diff
@@ -362,6 +374,14 @@ spec:
             items:
               - key: receptor_conf
                 path: receptor.conf
+{% if projects_persistence|bool %}
+        - name: "{{ ansible_operator_meta.name }}-projects"
+          persistentVolumeClaim:
+{% elif projects_existing_claim %}
+            claimName: {{ projects_existing_claim }}
+{% else %}
+            claimName: '{{ ansible_operator_meta.name }}-projects-claim'
+{% endif %}
```

Comment:
Simply, these should be:

```yaml
{% if projects_persistence|bool %}
        - name: "{{ ansible_operator_meta.name }}-projects"
          persistentVolumeClaim:
{% if projects_existing_claim %}
            claimName: {{ projects_existing_claim }}
{% else %}
            claimName: '{{ ansible_operator_meta.name }}-projects-claim'
{% endif %}
{% endif %}
```

We should test this block with following four patterns and your implementation generates corrupted yaml in all patterns 😢 

1.  `projects_persistence: true`, `projects_existing_claim: mypvc`
2.  `projects_persistence: true`, `projects_existing_claim: {}`
3.  `projects_persistence: false`, `projects_existing_claim: mypvc`
4.  `projects_persistence: false`, `projects_existing_claim: {}`